themes/default/theme/stencil/src/components/ai/client.ts

```diff
@@ -19,7 +19,7 @@ import { parseCookie } from "../../util/util";
 export class PulumiAIClient {
     private socket: WebSocket;
 
-    public MAX_PROMPT_LENGTH = 1024;
+    public MAX_PROMPT_LENGTH = 102400;
```

Comment:
I think this is too high of setting as a certain point the prompt will be bigger than the token limit of ChatGPT. `gpt-4` has a token limit of 8k and `gpt-3.5` has a token limit of 4k (a token is about 4 characters). So at most this could probably be ~24K, but we also still have to account for the fact that this input is injected into a larger prompt so we should constrain it a bit more.

Reply:
Ah, good to know, thanks. What do you think of, say, 10K then?

Reply:
I'll go ahead and do that -- we can come back to it if we notice any issues with it. Thanks again for the catch.